### ISSUE TITLE
Config files are not copied with the correct permissions

### ISSUE THREAD

**Issue by espenfl** [aiidalab/ansible-role-aiidalab-server#17]
Using Vagrant and `generic/ubuntu1804` the config files do not receive the correct permissions:

```
-rw-------   1 root root  4095 Aug 31 10:05 jupyterhub_config.py
(aiidalab) vagrant@ubuntu1804:/etc/jupyterhub$ 
```

They are also not explicitly set, here: https://github.com/aiidalab/ansible-role-aiidalab-server/blob/a5cc0074a6b2c32193df198b11479ad4924b709c/tasks/jupyterhub.yml#L13.

Suggest to update it. I will submit a PR.

**Comment by ltalirz** [comment]
thanks @espenfl , please go ahead.

indeed the whole `jupyterhub.yml` file is executed with `become: true`.
if this results in different file permissions on different operating systems, then we will indeed need to add specific user rights when copying files.

while you're at it, you're welcome to add centos / fedora to the tested operating systems on CI (just copying from the aiida role)

**Comment by yakutovicha** [comment]
fixed by #18
